plugin/www/mparticle.js

```diff
@@ -375,6 +391,52 @@ var mparticle = {
         'modify',
         [IdentityRequest])
     }
+  },
+
+  GDPRConsent: function () {
```

Comment:
nit: since these are identical objects, we could instead just merge them into a single `Consent` class and use that on both add methods